### DISCUSSION TITLE
Disable Windows Update Data Collection breaks Windows Updates

### DISCUSSION THREAD

**Discussion by ProgrammerBruce** [undergroundwires/privacy.sexy#461] [General]
I am pretty sure that one of the Disable OS Data Collection settings breaks Windows Updates and updates of the Network Status Icon (in the System Tray, which, yes, has another name).  Windows Updates attempts just result in Download Error.  When I have time, I will better narrow down which setting is the issue.

Is this expected of the Disable OS Data Collection set?

**Comment by ProgrammerBruce** [comment]
Reverting "Disable Windows Update data collection" resolved the Windows Updates Download Error.  Of the two parts therein, maybe the issue was caused just by "Disable Delivery Optimization Service".

Regarding the Network Status Icon update to show connectivity correctly, I see in "Disable connectivity checks" that "Remove Network Connectivity Status Indicator (NCSI) app (breaks Internet connection status icon)".

Issue logged: https://github.com/undergroundwires/privacy.sexy/issues/462

Changing this Discussion post title...

UPDATE: I confirmed that it was just the "Disable Delivery Optimization Service" that broke downloading of Windows Updates for Windows 11 24H2 build 26100.2454.  I do not know if the DoSvc can be stopped/started manually without restart.

**Reply by CandyClass** [reply]
Really saved my win11. Everything else I can find not work for me in these issue, though your method works. Precisely, I followed this path to find out my DoSvc service's 'start' key is set to 4 (disable), change to 2 or 3 then reboot fix the update problem.

**Comment by undergroundwires** [comment]
🚀 Clarified in [0.13.8](https://github.com/undergroundwires/privacy.sexy/releases/tag/0.13.8), see https://github.com/undergroundwires/privacy.sexy/issues/462#issuecomment-2721386341, thank you for the reporting @ProgrammerBruce .
